shapely/constructive.py

```diff
@@ -926,6 +927,56 @@ def simplify(geometry, tolerance, preserve_topology=True, **kwargs):
         return lib.simplify(geometry, tolerance, **kwargs)
 
 
+@requires_geos("3.11.0")
+@multithreading_enabled
+def simplify_polygon_hull(
+    geometry, parameter, parameter_mode="area", is_outer=True, **kwargs
+):
+    """Computes a boundary-respecting hull of a polygonal geometry, with hull
+    shape determined by a target parameter specifying the fraction of the input
+    vertices or the input geometry's area retained retained in the result.
+
+    Parameters
+    ----------
+    geometry : Geometry or array_like
+    parameter : float or array_like
+        Larger values produce less concave results. A value of 1 produces
+        the convex hull; a value of 0 produces the original geometry.
```

Comment:
This is actually correct only for area mode. In the vertex mode, it is the vice versa. 0 is a convex_hull, while 1 is original geometry (as in PostGIS). That can be a bit confusing. I would recommend keeping both the same, probably following the vertex mode and PostGIS.

Reply:
Also, it is convex hull only if `is_outer=True`, otherwise it is the largest triangle within the geom (I think).

Reply:
Yes, it's easy to misunderstand. The explanation should be:

Two parameters are supported:
- Vertext Mode: the tolerance means Vertex Number fraction. the fraction of the input vertices retained in the result. Value 1 produces the original geometry. Smaller values produce less concave results. For outer hulls, value 0 produces the convex hull (with triangles for any holes). For inner hulls, value 0 produces a triangle (if no holes are present). The value should be in the range [0,1].
- Area Mode: the tolerance means Area Delta ratio. the ratio of the change in area to the input area. Value 0 produces the original geometry. Larger values produce less concave results. The value must be 0 or greater.

Reply:
Parameter is in range [0, 1] for vertex mode and in range [0, ∞) for area mode. 
```python
a = shapely.Polygon([[0,0], [100,0], [200, 500], [100, 10], [0, 10]])
b = shapely.simplify_polygon_hull(a, 1, "area", True)
c =shapely.simplify_polygon_hull(a, 10, "area", True)
d = shapely.simplify_polygon_hull(a, 99, "area", True)
print(a.area, b.area, c.area, d.area) # 1500.0 1500.0 1500.0 26000.0
```
I can keep both same by limit the tolerance in the range [0, 1] like
```python
    if parameter_mode == "area":
        parameter_mode = 2
        tolerance = 1-tolerance if tolerance < 1 else 0
    elif parameter_mode == "vertex":
        parameter_mode = 1
```
However, it will limit result in area mode as the area delta ratio couldn't be larger than 1 in this case. It's hard to keep both same.😟